src/screens/TermsScreen/TermsScreen.tsx

```diff
@@ -3,25 +3,21 @@ import { TextInput, View } from 'react-native';
 import { ScrollView } from 'react-native-gesture-handler';
 import { Title, SpacerInline, NavBar } from '@elements';
 import Terms from '@assets/documents/SampleToS';
+import ScrollContainer from '../../elements/ScrollContainer/ScrollContainer';
 import styles from './TermsScreen.styles';
 
 export default () => (
 	<View style={styles.outerContainer}>
 		<NavBar showMenu={false} />
 		<Title text="Terms and conditions" />
 		<SpacerInline height={20} />
-		<ScrollView contentContainerStyle={{ justifyContent: 'space-between' }}>
-			<View style={styles.documentContainer}>
-				<TextInput
-					style={styles.documentText}
-					multiline={true}
-					editable={false}
-					scrollEnabled={true}
-				>
-					{Terms}
-				</TextInput>
-			</View>
-		</ScrollView>
+		<ScrollContainer
+			documentText={Terms}
+			displayCheckbox={true}
+			setChecked={() => {}}
```

Comment:
I think the linter prefers we pass the `Function` keyword instead of an empty arrow function, but no biggie either way.